compiler/src/dmd/frontend.h

```diff
@@ -8873,7 +8876,9 @@ struct Id final
     static Identifier* naked;
     static Identifier* thread;
     static Identifier* vector_size;
+    static Identifier* noinline;
     static Identifier* noreturn;
+    static Identifier* _deprecated;
```

Comment:
These seem unrelated to this PR.

Reply:
Yes, it should have gone with a previous PR of mine, but I didn't think I needed to update frontend.h.

Reply:
Well then, I guess the best way to move forward is to undo this change and then open a PR to update the headers?

Reply:
Seems like a waste of time.